.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
And for pip, this should be better: http://docs.travis-ci.com/user/caching/#pip-cache

Isn't it?